shared/chat/conversation/input-area/normal/index.js

```diff
@@ -100,18 +182,132 @@ class Input extends React.PureComponent<InputProps> {
     }
   }
 
+  _getUserSuggestions = filter => {
+    const fil = filter.toLowerCase()
+    return this.props.suggestUsers
+      .filter(
+        user => user.username.toLowerCase().includes(fil) || user.fullName.toLowerCase().includes(filter)
+      )
+      .toArray()
+  }
+
+  _renderUserSuggestion = ({username, fullName}: {username: string, fullName: string}, selected: boolean) => (
+    <Kb.Box2
+      direction="horizontal"
+      fullWidth={true}
+      style={Styles.collapseStyles([
+        styles.suggestionBase,
+        styles.fixSuggestionHeight,
+        {
+          backgroundColor: selected ? Styles.globalColors.blue4 : Styles.globalColors.white,
+        },
+      ])}
+      gap="small"
+    >
+      {!Constants.isSpecialMention(username) ? (
```

Comment:
I think it'd be clearer to avoid the negative test here, i.e. `Constants.isSpecialMention(...) ? ... : ...`